google-map-directions.html

```diff
@@ -53,8 +58,10 @@
        * @type google.maps.Map
        * @default null
        */
-      map: null,
-
+      map: {
+        type: Object,
+        observer: 'mapChanged'
```

Comment:
Yes. Turns out that leaving values undefined makes computed properties not fire. I think specifying default value is a must. Done.